devito/types/sparse.py

```diff
@@ -1974,11 +2175,11 @@ def _dist_scatter(self, data=None):
 
     # The implementation in AbstractSparseFunction now relies on us
     # having a .coordinates property, which we don't have.
-    def _arg_apply(self, dataobj, alias=None):
+    def _arg_apply(self, *dataobj, alias=None):
```

Comment:
I would rather have:

```
def _arg_apply(self, dataobj, *subfuncs, alias=None):
```

or `*args`; so that you don't have to use `[0]` below. Also, `dataobj` indicates the true Function data

Reply:
Had it like that in abstract one fixed here